### PR TITLE
HZN-151: Documentation for Page Sequence Monitor

### PR DESCRIPTION
**Pull request by indigo423**
Add documentation to allow an administrator to create a Page Sequence Monitor. Explain all available configuration parameters for the monitor and give useful examples.

JIRA: http://issues.opennms.org/browse/HZN-151
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS282

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+The overall layout of the monitor configuration is more complex.
+Additional to the parameter keys it is possible to configure a page sequence containing a path through an web application.
+
+.Monitor parameters for the PageSequenceMonitor
+[options="header, autowidth"]
+|===
+| Parameter        | Description                                                                           | Required | Default value
+| `timeout`        | Time in milliseconds to wait before assuming that a packet has not responded          | optional | `3000`
+| `retry`          | The number of retries to attempt when a packet fails to respond in the given timeout  | optional | `0`
+| `strict-timeout` | ?? `true` or `false`                                                                  | optional | `false`
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc]: @jeffgdotorg Do you have a description sentence for `strict-timeout` parameter setting?

File: opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+| `page-sequence`  | Definition of the page-sequence to execute, see table with _Page Sequence Parameter_  | required | `-`
+| `sequence-retry` | ??
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc]: @jeffgdotorg Do you have a description sentence for `sequence-retry` parameter setting?

File: opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+| `name`                     | The name of the page-sequence. *(Is this relevant/used?)*                                  | optional | `-`
+| `method`                   | _HTTP_ method for example _GET_ or _POST_                                                  | `-`      | `-`
+| `http-version`             | _HTTP_ protocol version number, 0.9, 1.0 or 1.1                                            | optional | `HTTP/1.1`
+| `user-agent`               | Set the _user agent_ field in _HTTP_ header to identify the OpenNMS monitor                | optional | `-`
+| `virtual-host`             | Set the _virtual host_ field in _HTTP_ header.
+                               In case of an HTTPS request, this is also the virtual domain to send as part of the _TLS_
+                               negotiation, known as server name indication (_SNI_)
+                               (See: link:https://www.ietf.org/rfc/rfc3546.txt[RFC3546] section 3.1)                      | `-`      | `-`
+| `path`                     | The relative URL to call in the request.                                                   | ??       | ??
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc]: We don't know the default value and if it is required.

File: opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+| `strict-timeout` | ?? `true` or `false`                                                                  | optional | `false`
+| `page-sequence`  | Definition of the page-sequence to execute, see table with _Page Sequence Parameter_  | required | `-`
+| `sequence-retry` | ??
+|===
+
+.Page Sequence Parameter
+[options="header, autowidth"]
+|===
+| Parameter                  | Description                                                                                | Required | Default
+| `name`                     | The name of the page-sequence. *(Is this relevant/used?)*                                  | optional | `-`
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc]: @jeffgdotorg Do you have a description sentence for the `page name` parameter setting?

**Comment by opennms-bamboo** [comment]
Welcome @xplorn.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @xplorn.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](http://www.opennms.org/documentation/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit http://www.opennms.org/wiki/Contributor_Agreement

**Comment by mvrueden** [comment]
Hey Ulf, please approve ken@opennms.org as xplorn.

**Comment by mvrueden** [comment]
Hey Ulf, can you please redo the OCA check?

**Comment by fooker** [comment]
As this was not merged in v17.0.0 please open another pull request against branch ```release-17.0.1```.

**Pull request closed without merging**